src/wave.js

```diff
@@ -73,14 +73,32 @@ class WaveBase {
         window.addEventListener('resize', () => {this.handleResize()}, false);
     }
 
-    initCamera() {
+    initOrthographicCamera() {
         const initialPosition = [-50, 0, 10];
-        this.camera = new THREE.PerspectiveCamera(20, this.ASPECT, 1, 20000);
-        this.camera.name = "PerspectiveCamera";
-        this.camera.position.copy(new TV3(...initialPosition));
-        this.camera.up = new TV3(0, 0, 1);
-        this.camera.lookAt(new TV3(0, 0, 0));
-        this.scene.add(this.camera);
+        this.orthographicCamera = new THREE.OrthographicCamera(-10 * this.ASPECT, 10 * this.ASPECT, 10, -10, 1, 1000);
+        this.orthographicCamera.name = "OrthographicCamera";
+        this.orthographicCamera.position.copy(new TV3(...initialPosition));
+        this.orthographicCamera.up = new TV3(0, 0, 1);
+        this.orthographicCamera.lookAt(new TV3(0, 0, 0));
+        this.scene.add(this.orthographicCamera);
```

Comment:
I'd move this to `initCamera` and call that in constructor - shall we?

Reply:
`initCamera` is divided into two parts, `initOrthographicCamera ` and `initPerspectiveCamera ` both called in the constructor.

Reply:
Well, that the confusing part - we only have one camera active in the scene (line 92) and it happens in one of inits but not the other - let's isolate that line to:

```js
initCameras() {
        this.initPerspectiveCamera();
        this.initOrthographicCamera();
        this.camera = this.perspectiveCamera;
}
```
and call `initCameras` in constructor?

also lines 79-81 and 88-90 are similar - let's move to `setCameraPosition(camera)` and also have 1, 0, 0 and 0, 0, 0 set in settings?

Reply:
Done.